### DISCUSSION TITLE
中文汉化

### DISCUSSION THREAD

**Discussion by jiaoTaiLang404** [upscayl/upscayl#844] [Q&A]
汉化包在哪下载

**Comment by aaronliu0130** [comment]
Duplicate of #91.
> I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

![compulsory rickrolls are issued for duplicates of issues and the FAQ](https://user-images.githubusercontent.com/11874211/218282922-efab6795-84af-4cfb-9400-c28c8729ea78.gif)
网上应该有
